packages/babel-core/src/tools/build-external-helpers.js

```diff
@@ -139,12 +139,16 @@ function buildVar(namespace, builder) {
 }
 
 function buildHelpers(body, namespace, whitelist) {
+  const getHelperReference = name =>
+    t.memberExpression(namespace, t.identifier(name));
+
   helpers.list.forEach(function(name) {
     if (whitelist && whitelist.indexOf(name) < 0) return;
 
     const { nodes } = helpers.get(
       name,
-      t.memberExpression(namespace, t.identifier(name)),
+      getHelperReference,
```

Comment:
We'll also need logic like this in `babel-runtime/scripts/build-dist.js` so `babel-runtime` has the correct imports and exports to link the files together.